recipes/bioconductor-txdb.hsapiens.ucsc.hg38.knowngene/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: bioconductor-txdb.hsapiens.ucsc.hg38.knowngene
+  version: 3.4.0
+
+
+# Keeping the sources in here should trigger any warnings about missing URLs or
+# hash mismatches at build time, even though nothing is actually done with the
+# downloaded package.
+source:
+  fn: TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz
+  url:
+    - http://bioconductor.org/packages/3.5/data/annotation/src/contrib/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0.tar.gz
+    - https://depot.galaxyproject.org/software/TxDb.Hsapiens.UCSC.hg38.knownGene/TxDb.Hsapiens.UCSC.hg38.knownGene_3.4.0_src_all.tar.gz
+  md5: 1d5e07631ea58e96b11905d39e76ca6e
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - bioconductor-annotationdbi
+    - 'bioconductor-genomicfeatures >=1.25.18'
+    - r-base
```

Comment:
If this package only installs a post-link script, why does it have build dependencies?